### DISCUSSION TITLE
[Script request] WGER - fitness/workout, nutrition and weight tracker

### DISCUSSION THREAD

**Discussion by deku-m** [community-scripts/ProxmoxVE#514] [Request script] [answered]
### Application Name

WGER

### Website

https://wger.de/nl/software/features

### Description

wger (ˈvɛɡɐ) Workout Manager is a free, open source web application that helps you manage your personal workouts, weight and diet plans and can also be used as a simple gym management utility. It offers a REST API as well, for easy integration with other projects and tools.

For a live system, refer to the project's site: https://wger.de/

### Due Diligence

- [X] I have searched existing [scripts](https://community-scripts.github.io/Proxmox/scripts) and found no duplicates.
- [X] I have searched existing [discussions](https://github.com/community-scripts/ProxmoxVE/discussions?discussions_q=) and found no duplicate requests.
- [X] This is not a game-related request.

**Comment by tremor021** [comment, marked as answer]
Done in #2574
